### DISCUSSION TITLE
color issue on Mac terminal

### DISCUSSION THREAD

**Discussion by zhaob1n** [helix-editor/helix#2276] [Q&A] [answered]
I installed helix through home-brew and when I enter hx to open helix, the color shows like this. Why and what should I do
<img width="579" alt="Screen Shot 2022-04-26 at 12 27 55 PM" src="https://user-images.githubusercontent.com/52552971/165221346-00c56909-ea2e-4ef7-8ceb-562d980c43ca.png">
?

**Comment by the-mikedavis** [comment, marked as answer]
https://github.com/helix-editor/helix/wiki/Troubleshooting#rendering-issues-on-macos-terminal

> The MacOS terminal lacks true color support, so you'll need to install a terminal that has it.

The default MacOS terminal can't render true colors. Instead you should try a terminal like iTerm2, Kitty, Alacritty, or WezTerm.
